lib/rules/sotd/obsl-rescind.js

```diff
@@ -0,0 +1,53 @@
+// SotD
+// <p>W3C has chosen to (rescind|obsolete) the Sample Specification Recommendation for the
+// following reasons: [...list of reasons...]. For additional information about
+// replacement or alternative technologies, please refer to the <a
+// href="https://www.w3.org/2016/11/obsoleting-rescinding/">explanation of
+// Obsoleting and Rescinding W3C Specifications</a>.</p>
```

Comment:
Just curious: where does this copy come from? I can't find it on [`w3.org/2015/Process-20150901`](https://www.w3.org/2015/Process-20150901/#rec-rescind) nor on [`w3c.github.io/w3process-obsolete-rescinded`](https://w3c.github.io/w3process-obsolete-rescinded/).